import exceptions

print(__name__)

print(exceptions.greet('Student'))